first-30-minutes-of-the-writing-task-in-studieproeven.md

```diff
@@ -32,18 +32,18 @@ To write a coherent essay, you need to understand the context of the Writing tas
 
 1. Read and make sure that you have understood the explanatory sentences under the data material first (e.g., a table or the diagram) before looking at the data set. They often contain food for thought and hints towards what to pick up for data description. In other words, they cast the light on the issue and give the contextual and non-statistical summary of the issue. The explanatory sentences look like this in Opgave 1 ‘Land og By’ from winter 2019 (see under the to tables): 
 
-Explanatory sentences
+   Explanatory sentences:
```

Comment:
These paragraphs belong to the "1." bullet? Then they should be intended accordingly.